src/main.rs

```diff
@@ -17,13 +17,28 @@ use config::Config;
 
 /// Tiny CLI utility to checkout a recent git branch interactively.
 fn main() -> Result<()> {
-    match run_tui() {
-        Ok(()) => Ok(()),
-        Err(e) => match e.downcast_ref::<SelectBranchError>() {
-            Some(SelectBranchError::Aborted) => process::exit(1),
-            Some(SelectBranchError::Interrupted) => process::exit(2),
-            None => Err(e),
+    let args: Vec<String> = env::args().collect();
+    match args.len() {
+        1 => match run_tui() {
+            Ok(()) => Ok(()),
+            Err(e) => match e.downcast_ref::<SelectBranchError>() {
+                Some(SelectBranchError::Aborted) => process::exit(1),
+                Some(SelectBranchError::Interrupted) => process::exit(2),
+                None => Err(e),
+            },
         },
+        2 => {
+           if ( args[1] == "--version" ) | ( args[1] == "-V" ) {
```

Comment:
In my experience, a lower-case "v" is the more common short-hand (though to be fair, I almost exclusively use unix-systems). I wouldn't be opposed to making the flags fully case insensitive if you prefer that?

```suggestion
           if ( args[1] == "--version" ) | ( args[1] == "-v" ) {
```